apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/backend/codegen/FragmentType.kt

```diff
@@ -71,12 +83,5 @@ internal fun CodeGenerationAst.FragmentType.implementationTypeSpec(generateAsInt
       .toBuilder()
       .addSuperinterface(GraphqlFragment::class)
       .applyIf(generateAsInternal) { addModifiers(KModifier.INTERNAL) }
-      .addType(TypeSpec.companionObjectBuilder()
-          .addProperty(PropertySpec.builder("FRAGMENT_DEFINITION", String::class)
```

Comment:
👍 I guess we don't need that ?

Reply:
Yep, for named fragment default implementation we don't need graphql definition as it already declared in interface